### DISCUSSION TITLE
Added a text-xxs font size to my tailwind.config but it overrides colors - how to fix?

### DISCUSSION THREAD

**Discussion by stevensacks** [dcastil/tailwind-merge#489] [Help] [answered]
I added this to my theme in tailwind.config
```ts
fontSize: {
  xxs: '0.6875rem',
},
```
But twMerge causes it to replace colors.
```ts
twMerge('text-white', 'text-xxs')

// outputs
'text-xxs'
```
Tailwind Merge thinks it's a color.

I tried reading the docs about limitations an configurations, but I haven't been able to figure it out.

Is there a way to configure twMerge to understand that text-xxs is a font size and not a color?

**Comment by dcastil** [comment, marked as answer]
Hey @stevensacks! 👋

tailwind-merge doesn't have access to the tailwind.config.js file and you need to configure it separately so it knows about the text-xxs class. But tailwind-merge will recognize it automatically if you use `text-2xs` or `text-0.5xs` (similar to the class `text-2xl`) in case you prefer that.

Here is an example on how to configure tailwind-merge: https://github.com/dcastil/tailwind-merge/blob/v2.5.4/docs/recipes.md#adding-custom-scale-from-tailwind-config-to-tailwind-merge-config.

And here is the documentation on how the tailwind-merge configuration works: https://github.com/dcastil/tailwind-merge/blob/v2.5.4/docs/configuration.md#usage-with-custom-tailwind-config.

---

(For myself)

Related: https://github.com/dcastil/tailwind-merge/issues/469, https://github.com/dcastil/tailwind-merge/issues/447, https://github.com/dcastil/tailwind-merge/issues/368, https://github.com/dcastil/tailwind-merge/discussions/322, https://github.com/dcastil/tailwind-merge/issues/321, https://github.com/dcastil/tailwind-merge/discussions/315, https://github.com/dcastil/tailwind-merge/issues/302, https://github.com/dcastil/tailwind-merge/issues/276, https://github.com/dcastil/tailwind-merge/discussions/275, https://github.com/dcastil/tailwind-merge/issues/274, https://github.com/dcastil/tailwind-merge/issues/250, https://github.com/dcastil/tailwind-merge/issues/207

**Reply by stevensacks** [reply]
Thanks! Changing it to `text-2xs` worked!
